src/main/java/space/obminyashka/items_exchange/controller/AuthController.java

```diff
@@ -69,9 +69,10 @@
     @PostMapping("/logout")
     @ApiOperation(value = "Log out a registered user")
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void logout(HttpServletRequest req, HttpServletResponse resp, Authentication auth, @RequestHeader(HttpHeaders.AUTHORIZATION) String token) {
-        new SecurityContextLogoutHandler().logout(req, resp, auth);
-        if (!authService.logout(token, auth.getName())) {
+    public void logout(HttpServletRequest req, HttpServletResponse resp, @ApiIgnore Authentication authentication,
```

Comment:
Why don't you line each of those variables from the next line?

Reply:
It'll really look better this way. Thanks.